opencog/nlp/relex2logic/rules/tense.scm

```diff
@@ -0,0 +1,64 @@
+;Both link grammar and relex output are selected because of the pattern 
+;Example : "He gave me the book."
+;(DefinedLinguisticConceptNode "past") and (DefinedLinguisticConceptNode ".v-d")
+;I just want to avoid the Link Grammar output. There might be another way to do this
+(define (check-tense tense)
+    (if (string-contains (cog-name tense) ".")
```

Comment:
No, But I just checked it now (http://wiki.opencog.org/w/AbsentLink) . From the explanation on that page it doesn't seems a solution for this case.
Because here  two DefinedLinguisticConceptNode 's are present in the atomspace… one from link grammar and the other from relex and they are selected because of there pattern. I want the pattern matcher not to select the DefinedLinguisticConceptNode  from link grammar